lib/openstates/models/bill.rb

```diff
@@ -32,7 +32,7 @@ def api_method
       end
 
       def id_key
-        :bill_id
+        :id
```

Comment:
I changed because the actual OpenState's id for the bill is the ```id``` attribute. The bill_id is the official bill id, which I can't use to fetch the bill on their system.